### PR TITLE
Added unity cheatsheet

### PR DESCRIPTION
**Pull request by MathewtheCoder**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [x] Cheat Sheets: **`New Unity 5 Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

Unity 5 hotkeys

---

https://duck.co/ia/view/unity_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/unity.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+                "val": "Vertex Snap"
+            },
+            {
+                "key": "CTRL/CMD+LMB",
+                "val": "Snap"
+            }
+        ],
+        "GameObject": [
+            {
+                "key": "CTRL/CMD+SHIFT+N",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/unity.json]: For key combinations follow this kind of approach

[ Ctrl ] / [ Cmd ] [ Shift ] [ N ]

Comment (MathewtheCoder) [inline share/goodie/cheat_sheets/json/unity.json]: I made the change you asked for

File: share/goodie/cheat_sheets/json/unity.json [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+            {
+                "key": "[CTRL]/[CMD][D]",
+                "val": "Duplicate"
+            },
+            {
+                "key": "[SHIFT][Del]",
+                "val": "Delete"
+            },
+            {
+                "key": "[F]",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/unity.json]: remove brackets in case of only  one key

File: share/goodie/cheat_sheets/json/unity.json [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+            {
+                "key": "[CTRL]/[CMD][P]",
+                "val": "Play"
+            },
+            {
+                "key": "[CTRL/CMD][SHIFT][P]",
+                "val": "Pause"
+            },
+            {
+                "key": "[CTRL/CMD][ALT][P]",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/unity.json]: In keyboard shortcut cheat sheets we usually use 
Ctrl , Alt, Shift etc.
So lets avoid using all uppercase letters for keys

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_cheat_sheet_mobile.png?nxJGElkxHC)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_cheat_sheet_mobile.png?ttUSI3tJKX)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_cheat_sheet_mobile.png?Xvz4ENMyE0)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_cheat_sheet_mobile.png?o6XuV4pf71)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_cheat_sheet_mobile.png?fM8TAsRuKX)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/unity_cheat_sheet_mobile.png?XFQoL7Ui5G)

File: share/goodie/cheat_sheets/json/unity.json [hunk 4]
```diff
@@ -0,0 +1,183 @@
+{
+    "id": "unity_cheat_sheet",
+    "name": "Unity 5",
+    "description": "Unity HotKeys",
+
+    "metadata": {
+        "sourceName": "Unity",
+        "sourceUrl" : "http://docs.unity3d.com/Manual/UnityHotkeys.html"
```
Comment (122131) [inline share/goodie/cheat_sheets/json/unity.json]: I think HTTPS is better here 👍

**Comment by aiyankovil** [comment]
@MathewtheCoder pls close the pull request and put a new PR. I have faced the same problem"This branch has conflicts that must be resolved".its because your master branch is behind the original.
git checkout master
git reset --hard upstream/master
git checkout -b unity-cheatsheet
** add your JSON file **
git status (everything looks good?)
git commit -am "New Unity cheat sheet"
git push
** open a new pull request **

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**